### DISCUSSION TITLE
GPS Sensor User Interface

### DISCUSSION THREAD

**Discussion by triatebr** [BIPES/BIPES#185] [General]
When clicking on the GPS sensor, all the options appear piled on top of each other, making it very difficult to use the GPS.
Can someone help me?
I do not know what else to do
It only happens with this sensor, the others are ok

GPS: [link](https://drive.google.com/file/d/18LNL2Sc32FuYb31R-e9jdByz5vrPRTta/view?usp=sharing)
Oximeter: [link](https://drive.google.com/file/d/1XaYZ32RVwzQK8BTcBqtuhY6KNeCbWi4Y/view?usp=sharing)

the correct interface should look like this [link](https://drive.google.com/file/d/1F8lVsHp-SN-QGOs7tMG9dA8j8RXENwg1/view?usp=sharing)

I ask for help
thanks

**Comment by bsimmo** [comment]
same here, was going to test it out to try the bipes block setup but i'll come back later.

**Comment by jetannenbaum** [comment]
Sorry, thought I replied to this earlier.  I updated the offline branch as well as the pico online branch https://bipes.net.br/pico/ui/ to properly show support for the GPS sensor for the RP2040 microcontrollers.  I do not have the sensor and I do not have any other processors, so support was not updated for other platforms.  I also do not support the main branch, since my changes are only tested for boards I own.  If you can test, I would be happy to try to support any issues you find.  Thanks.

**Reply by jetannenbaum** [reply]
Found the update I made previously.  I replied to the OP post about the import failing for the library.
